test/functional/basic/basic-e2e-specs.js

```diff
@@ -26,6 +26,18 @@ describe('XCUITestDriver - basics', function () {
       let status = await driver.status();
       status.wda.should.exist;
     });
+
+    it('should cache status if another operation is in progress', async () => {
+      await driver.setImplicitWaitTimeout(20000);
+
+      let result;
+      B.Promise.any(
+        [new B(driver.elementsByClassName)('WrongLocator'), new B(driver.status)()]
```

Comment:
No need for creating new promises. Without `await` both `driver.elementsByClassName()` and `driver.status()` return promises already.Also, does `B.any`(no need for the `Promise`) guarantee that the first element in the array is called first? I can't find anything indicating that. If it does not, then sometimes this would not test what it is supposed to test.